src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -126,6 +150,7 @@ public void sendSimpleMessage(TestContext context) {
                     JsonObject metadata = offsets.getJsonObject(0);
                     context.assertEquals(0, metadata.getInteger("partition"));
                     context.assertEquals(0L, metadata.getLong("offset"));
+                    context.assertEquals(HttpResponseStatus.OK.code(), response.statusCode());
```

Comment:
I would move it up as first assertion. Please do the same for all the other places.

Reply:
right, it sounds reasonable, let me do it